packages/date-fns/src/date-fns-utils.ts

```diff
@@ -28,10 +28,10 @@ import startOfWeek from "date-fns/startOfWeek";
 import startOfYear from "date-fns/startOfYear";
 
 // Locale is not exported from date-fns, so we need to workaround that https://github.com/date-fns/date-fns/issues/932
-import SampleLocale from "date-fns/locale/en-US";
+import { enUS } from "date-fns/locale";
```

Comment:
I am aware of direct imports. Please check if it possible to use `date-fns/locale/enUS`